aws/table_aws_ecs_service.go

```diff
@@ -104,6 +104,11 @@ func tableAwsEcsService(_ context.Context) *plugin.Table {
 				Description: "The platform version on which to run your service.",
 				Type:        proto.ColumnType_STRING,
 			},
+			{
```

Comment:
Can you please move this above `platform_version` to help keep the columns in alphabetical order?